src/NUnitFramework/framework/Constraints/PredicateConstraint.cs

```diff
@@ -52,7 +52,7 @@ public override string Description
             get
             {
                 var name = predicate.GetMethodInfo().Name;
-                return name.StartsWith("<")
+                return name.StartsWith("<", StringComparison.Ordinal)
```

Comment:
I guess we could use the char-overload here?
```suggestion
                return name.StartsWith('<')
```